config.yml

```diff
@@ -16,37 +16,38 @@ params:
     "Boards": ""
     "Conferences": ""
     "Cores": ""
-    "Cores:Collection": "collections containing multiple IP cores"
-    "Cores:Library": "libraries containing multiple IP cores"
-    "Cores:Single": "single IP core"
+    "Cores:Collection": "Groups of individual IP cores"
+    "Cores:Library": "Strong-related set of IP cores"
+    "Cores:Single": "Single IP cores"
     "Frameworks": ""
-    "Frameworks:Verification": "frameworks for verification"
+    "Frameworks:Verification": "Frameworks for verification"
```

Comment:
Other types of frameworks to be considered?

Reply:
E.g. frameworks for "Internet of Things" including software and hardware modules that work together.